entry/src/main/js/MainAbility/pages/index/index.css

```diff
@@ -0,0 +1,118 @@
+.container1{
+    display:flex;
+    flex-direction: column;
+    background-color:#2E2E2E ;
+    align-items: center;
+    border-radius:40px;
+}
+.child-container1{
+    background-image: url("/common/images/indication_bg.png");
```

Comment:
please format the code, use Ctrl + Alt + L shortcut for the same in the studio editor.